### ISSUE TITLE
Recruitment: 3rd Party Bounty Postings

### ISSUE THREAD

**Issue by Keyrxng** [ubiquity/.github#118]
## What is it?

Similar to [our DoraHacks bounty post](https://github.com/ubiquity/.github/issues/116) we should leverage similar channels and create a similar post/structure. Primarily it acts as just advertising space they are not considered full hackathons or anything remotely close. 

We should stick to the same format and just call them something like `Newcomers Bonus` and have it follow the same pattern where it's a bonus on top of your first task.

## Where would we post?

- https://earn.superteam.fun/
- https://bounties.network/
- add more here

## Analytics

- we can perform the same test conditions across all of them then consider dropping poor performing ones from any future usage of this strategy

## What's required?

- No uniform or automated way of handling submissions (as of yet) as each platform handles things in their own way so would require manually handling things until we can automate/improve our end. 
- It likely requires minimal effort probably a quick check of the site(s) once per week and validating submissions

#### STILL TO-DOS:

- find more places to post/validate their usefulness i.e userbase, social proofs etc
- either improve the current DoraHacks bounty post and we'll replicate it across all postings, use it as-is or create a varied post for each platform.

Creating a slightly different post for each platform might be a great idea for SEO

**Comment by Keyrxng** [comment]
I have intentionally not included big names that would require a substantial bounty prize to look attractive. What I mean by that is there are a lot more platforms that offer similar things but the pools are like $5k-$50k and are more like traditional Bug Bounty programs. There are blue chips amongst a lot of others so it's mixed.

The problem isn't so much the trad bug bounty theme as we could spin anything really into a time-based bonus event. I.e we could create labels for particular tasks only for the duration of the "bug bounty" event and they'd qualify for the bonus but they'd still see all of the other tasks and potentially return for them later

It's the value for money/cost-effectiveness of these attempts as I don't have solid data or anything we are just sort of winging it and seeing what sticks

**Comment by 0x4007** [comment]
My only concern is incentivizing sybils i.e. `newcomer bonus` but for limited time stuff I think its okay. Of course we want the most elite people but if the costs are slim/none then we should post everywhere, including places that may have less experienced developers.

**Comment by Keyrxng** [comment]
I agree on the matter of sybils which I think is near resolved after https://github.com/ubiquibot/plugins-wishlist/issues/26 as we could use the markers like `accountAge` and things of that nature to guard at the task level and for claiming the bonus through the 3rd party platform the submissions are only valid for one github account + platform account. Still possible but less so I guess

> We have https://github.com/ubiquity/business-development/issues/27#issuecomment-1617136647, some of which are valid

Great I hadn't seen this. I had passed up on a lot of them included here for reasons above cost, talent quality etc but I'll use what I can from it.

> Of course we want the most elite people but if the costs are slim/none then we should post everywhere

Using the $1k between 10 over 3 months would cost the same as a full-scale hackathon in terms of prize pool and I ask myself which would we see more quality from?

Let's set a max for how many of those tests we can run and then filtering the list for posts might be a bit easier

Any platform where the costs are nil we'll post

**Comment by 0x4007** [comment]
Budgets are generous for recruiting so let me know what you need when you need.
